packages/webpack-graph-ql/src/lib/types/warning.ts

```diff
@@ -0,0 +1,41 @@
+import { AutoSchema } from './auto-schema';
+
+type WarningData = {
+  message: string;
+  moduleId: string;
+  moduleIdentifier: string;
+  moduleName: string;
+  loc: string;
+  name: string;
+  warning: string;
+};
+
+// Resolvers for the "warning" field
+export class Warning extends AutoSchema {
```

Comment:
I think error, warning, and maybe a couple of others have enough common atrributes there should be an additional ancestor class to provide them